modules/appeals_api/app/serializers/appeals_api/higher_level_review_serializer.rb

```diff
@@ -4,6 +4,7 @@ class AppealsApi::HigherLevelReviewSerializer
   include FastJsonapi::ObjectSerializer
   set_key_transform :camel_lower
   attributes :status, :updated_at, :created_at
-  attribute :form_data, if: proc { |_record, options| !options[:is_collection] }
+  # only return form_data for created records
+  attribute :form_data, if: proc { |record| record.saved_change_to_id? }
```

Comment:
Let's use `record.new_record?` here to check whether it's a newly-created record or not. `saved_change_to_*` works, but it's a bit more esoteric of a check IMO.

Reply:
Yep I totally agree, but,  in this context is we save the new record before we serialize it, and new_record? only works for new unsaved records.
If you want, i could change the controller flow around to serialize and save to a var, then save, then return the var.

Reply:
hmm, nah, I didn't realize it the `new_record?` wouldn't apply here. Keep it with the `saved_change_to_id?`